docs/pages/inboxes/disappearing-messages.md

```diff
@@ -0,0 +1,84 @@
+# Support disappearing messages with XMTP
+
+Disappearing messages are messages that are intended to be visible to users for only a short period of time. After the message expiration time passes, the messages are removed from the UI and deleted from local storage so the messages are no longer accessible to conversation participants.
+
+Disappearing message behavior is enforced by apps, meaning that apps are responsible for removing messages from their UIs and local storage based on conditions set at the conversation level. Conversation participants using apps that support disappearing messages will have a UX that honors the message expiration conditions. 
+
+However, it's important to understand that:
+
+- A conversation participant using an app that doesn't support disappearing messages won't experience disappearing message behavior.
+- Messages aren't deleted from the XMTP network.
+
+:::tip
+
+Therefore, disappearing messages should be understood as best-effort, app-level privacy that helps avoid leaving an easily accessible record in a messaging UI. However, it's not a guaranteed, system-wide “message self-destruct.”
+
+:::
+
+## Implement disappearing messages
+
+### Enable disappearing messages for a conversation
+
+When creating or updating a conversation, admins can set disappearing message expiration conditions.
```

Comment:
@mchenani Did I capture your feedback correctly here? Specifically, only admins can set this?

Would it be helpful to even more specifically say:

When creating or updating a conversation, only group admins and DM participants can set disappearing message expiration conditions.

Reply:
That's all correct, and I like the second suggestion since it's covering the DMs as well.